tooling-docker/src/test/java/org/jboss/eap/qe/ts/common/docker/DockerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.jboss.eap.qe.ts.common.docker;
+
+import io.restassured.RestAssured;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static org.hamcrest.Matchers.containsString;
+
+public class DockerTest {
+
+    @ClassRule
+    public static Docker wildflyContainer = new Docker.Builder("wildfly", "jboss/wildfly:18.0.0.Final")
+            .setContainerReadyTimeout(60000)
+            .setContainerReadyCondition(() -> {
+                try {
+                    URL url = new URL("http://127.0.0.1:9990/health");
+                    HttpURLConnection con = (HttpURLConnection) url.openConnection();
+                    con.setRequestMethod("GET");
+                    if (con.getResponseMessage().contains("OK")) {
+                        return true;
+                    }
+                } catch (Exception ex) {
+                    // exceptions means not ready
+                    return false;
+                }
+                return false;
+            })
+            .wihtPortMapping("8080:8080")
+            .wihtPortMapping("9990:9990")
+            .withCmdArg("/opt/jboss/wildfly/bin/standalone.sh")
+            .withCmdArg("-b=0.0.0.0")
+            .withCmdArg("-bmanagement=0.0.0.0")
+            .build();
+
```

Comment:
Can we also have Jaeger included here ?

So 2 docker containers are running + 2 tests are here (One for WF, one for Jaeger).

Reply:
I did not use Jaeger intentionally as it takes over a minute for Jaeger container to start which is much longer I'd like to see spent time on testing our tooling. 

I understand the point to test 2 docker containers started. What about to use just https://hub.docker.com/_/hello-world ?

Reply:
+1